.github/pull_request_template.md

```diff
@@ -7,4 +7,4 @@
 <!-- If you're fixing an issue with this pull request, use the Fixes keyword with the issue number you're closing, like this:
 
 Fixes #1
--->
\ No newline at end of file
+-->
```

Comment:
Prettier is adding a new line at the end here.

Reply:
Files should end with a newline anyways, so that there's "room" to grow in the file. 😄